lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -172,10 +179,13 @@ export const migrateGranuleAndFilesViaTransaction = async (
     } else {
       granulesSummary.failed += 1;
       filesSummary.failed += files.length;
-      logger.error(
-        `Could not create granule record and file records in RDS for DynamoDB Granule granuleId: ${dynamoRecord.granuleId} with files ${dynamoRecord.files}`,
-        error
-      );
+      errorMessage = `Could not create granule record and file records in RDS for DynamoDB Granule granuleId: ${dynamoRecord.granuleId} with files ${JSON.stringify(dynamoRecord.files)}`;
+      errorFile.push(`Error: ${error} ${errorMessage}`);
+      logger.error(errorMessage, error);
+    }
+
+    if (bucket) {
```

Comment:
Out of curiosity: is there any way once this file is written that the user will easily be able to find groups of "related" errors? Such as "X number of granules failed bc of this missing collection", etc. 

Is it easy enough to somehow parse that information from the generated report? 

I think it might be possible to generate a report that "classifies" the errors by cause by throwing/catching specific error types, but not sure if that's overkill